<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1100</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>NetSniffer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QFrame" name="MainGrid">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>70</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="StartCapture">
            <property name="minimumSize">
             <size>
              <width>45</width>
              <height>45</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>45</width>
              <height>45</height>
             </size>
            </property>
            <property name="palette">
             <palette>
              <active>
               <colorrole role="Button">
                <brush brushstyle="SolidPattern">
                 <color alpha="0">
                  <red>255</red>
                  <green>255</green>
                  <blue>255</blue>
                 </color>
                </brush>
               </colorrole>
               <colorrole role="Base">
                <brush brushstyle="SolidPattern">
                 <color alpha="0">
                  <red>255</red>
                  <green>255</green>
                  <blue>255</blue>
                 </color>
                </brush>
               </colorrole>
               <colorrole role="Window">
                <brush brushstyle="SolidPattern">
                 <color alpha="0">
                  <red>255</red>
                  <green>255</green>
                  <blue>255</blue>
                 </color>
                </brush>
               </colorrole>
              </active>
              <inactive>
               <colorrole role="Button">
                <brush brushstyle="SolidPattern">
                 <color alpha="0">
                  <red>255</red>
                  <green>255</green>
                  <blue>255</blue>
                 </color>
                </brush>
               </colorrole>
               <colorrole role="Base">
                <brush brushstyle="SolidPattern">
                 <color alpha="0">
                  <red>255</red>
                  <green>255</green>
                  <blue>255</blue>
                 </color>
                </brush>
               </colorrole>
               <colorrole role="Window">
                <brush brushstyle="SolidPattern">
                 <color alpha="0">
                  <red>255</red>
                  <green>255</green>
                  <blue>255</blue>
                 </color>
                </brush>
               </colorrole>
              </inactive>
              <disabled>
               <colorrole role="Button">
                <brush brushstyle="SolidPattern">
                 <color alpha="0">
                  <red>255</red>
                  <green>255</green>
                  <blue>255</blue>
                 </color>
                </brush>
               </colorrole>
               <colorrole role="Base">
                <brush brushstyle="SolidPattern">
                 <color alpha="0">
                  <red>255</red>
                  <green>255</green>
                  <blue>255</blue>
                 </color>
                </brush>
               </colorrole>
               <colorrole role="Window">
                <brush brushstyle="SolidPattern">
                 <color alpha="0">
                  <red>255</red>
                  <green>255</green>
                  <blue>255</blue>
                 </color>
                </brush>
               </colorrole>
              </disabled>
             </palette>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton

{
		background-color: rgba(255, 255, 255, 0);
}
QPushButton:hover
{
	background-color: rgb(165, 165, 204);
	border-radius: 10px
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>128</height>
             </size>
            </property>
            <property name="MImage" stdset="0">
             <pixmap/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="StopCapture">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>45</width>
              <height>45</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>45</width>
              <height>45</height>
             </size>
            </property>
            <property name="sizeIncrement">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
		background-color: rgba(255, 255, 255, 0);
}
QPushButton:hover
{
	background-color: rgb(165, 165, 204);
	border-radius: 10px
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="iconSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Settings">
            <property name="minimumSize">
             <size>
              <width>45</width>
              <height>45</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>45</width>
              <height>45</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
		background-color: rgba(255, 255, 255, 0);
}
QPushButton:hover
{
	background-color: rgb(165, 165, 204);
	border-radius: 10px
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="iconSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="OpenPcap">
            <property name="minimumSize">
             <size>
              <width>45</width>
              <height>45</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>45</width>
              <height>45</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
		background-color: rgba(255, 255, 255, 0);
}
QPushButton:hover
{
	background-color: rgb(165, 165, 204);
	border-radius: 10px
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="iconSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="PrevPacket">
            <property name="minimumSize">
             <size>
              <width>45</width>
              <height>45</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>45</width>
              <height>45</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
		background-color: rgba(255, 255, 255, 0);
}
QPushButton:hover
{
	background-color: rgb(165, 165, 204);
	border-radius: 10px
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="NextPacket">
            <property name="minimumSize">
             <size>
              <width>45</width>
              <height>45</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>45</width>
              <height>45</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
		background-color: rgba(255, 255, 255, 0);
}
QPushButton:hover
{
	background-color: rgb(165, 165, 204);
	border-radius: 10px
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>10</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLineEdit" name="filterExpr"/>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QWidget" name="verticalLayoutWidget">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QFrame" name="frame">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <item row="0" column="0">
               <widget class="QTableView" name="PacketView"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="horizontalLayoutWidget">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QFrame" name="BottomFrame">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <widget class="QTabWidget" name="ProtoDetailWidget">
              <property name="geometry">
               <rect>
                <x>585</x>
                <y>10</y>
                <width>0</width>
                <height>31</height>
               </rect>
              </property>
              <property name="maximumSize">
               <size>
                <width>0</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="currentIndex">
               <number>1</number>
              </property>
              <widget class="QWidget" name="tab">
               <attribute name="title">
                <string>Tab 1</string>
               </attribute>
              </widget>
              <widget class="QWidget" name="tab_2">
               <attribute name="title">
                <string>Tab 2</string>
               </attribute>
              </widget>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="m_file">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="m_edit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="m_view">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <addaction name="m_file"/>
   <addaction name="m_edit"/>
   <addaction name="m_view"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
